### ISSUE TITLE
Unsatisfactory Dynamic and Static Separation Results

### ISSUE THREAD

**Issue by Chjx395** [chengweialan/DeSiRe-GS#7]
Dear Author,

Thank you for providing such user-friendly code. However, I encountered some difficulties when attempting to reproduce the dynamic-static separation results from your paper using the provided code. Could you clarify whether there are any details from the paper that are not explicitly implemented in the code? 

Specifically, I tried performing dynamic-static separation based on the script `separate.py`. Since the code only provides rendering for the static part, I wrote the following code for the dynamic part by following the structure of the static rendering implementation::

```
# Provided in the code
render_pkg_static = renderFunc(viewpoint, scene.gaussians, *renderArgs, env_map=env_map, mask=mask) 
# My addition
render_pkg_dynamic = renderFunc(viewpoint, scene.gaussians, *renderArgs, env_map=None, mask=~mask) 
```

**To ensure the validity of my experiments, I used the checkpoint provided in your repository.**  My results: 
![image](https://github.com/user-attachments/assets/cad21050-05eb-4b21-afb6-c98091d34172)

Results from the paper (another frame):
![image](https://github.com/user-attachments/assets/9ba55b37-9bb1-4211-9fdd-303bf448ddc0)

Clearly, the separation of the dynamic part in my results is not as clean as the results shown in the paper, though considering different frames.

Moreover, I noticed that the dynamic-static separation in the code is based on a custom threshold. I adjusted the default threshold of 0.2 to 0.1 and 0.3, and the results are shown below. 

Threshold = 0.1:
![351b226e-b060-43fd-92c2-dfc8991b43dc](https://github.com/user-attachments/assets/bebdbad9-fb39-45af-89ec-42502d9e87e3)

Threshold = 0.3:
![image](https://github.com/user-attachments/assets/ee6027a5-d5ec-457b-a435-bc95c6b26a2d)

Regardless of the threshold value, the separation of either the static or the dynamic part remains less clean compared to the results in your paper. Could you kindly provide any advice or insight into this issue?

**Comment by chengweialan** [comment]
Thank you for your interest in our work and for providing detailed feedback and examples.

As our method is fully self-supervised, the separation results may not be as accurate as those obtained using bounding boxes. Since we generated relatively accurate dynamic masks in the first stage, we used these masks to extract the dynamic components (the results in our paper).

We are also experimenting with using `v` instead of `scaling_t` for dynamic-static decomposition (or a combination of both). Feel free to try that as well!
